contents/graham_scan/code/c++/graham_scan.cpp

```diff
@@ -0,0 +1,65 @@
+#include <iostream>
+#include <vector>
+#include <algorithm>
+
+struct point{
+  double x;
+  double y;
+};
+
+bool ccw(point a, point b, point c){
+  return ((b.x - a.x)*(c.y - a.y) >= (b.y - a.y)*(c.x - a.x));
```

Comment:
This is not a __*`strict weak ordering`*__ Predication.
```suggestion
  return ((b.x - a.x)*(c.y - a.y) > (b.y - a.y)*(c.x - a.x));
```